README.md

```diff
@@ -21,46 +21,11 @@ This App is meant to be a drop-in add-on that can be installed with default conf
 The app will be running as a daemonset alongside the already existing `coreDNS` app, dramatically improving DNS resolution
 performances with a very small footprint in terms of system resources.
 
-## Network Policies
-
-When installing `k8s-dns-node-cache-app` you need to ensure any other workload running in the cluster has access to the new DNS service.
-This is an example network policy you might need to create:
-
-```
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
-metadata:
-  name: allow-node-local-dns-for-my-app
-spec:
-  podSelector:
-    matchLabels:
-      app: web
-  policyTypes:
-    - Egress
-  egress:
-  - to:
-    - ipBlock:
-        # this is the coredns service's Cluster IP.
-        cidr: 172.31.0.10/32
-    ports:
-    - protocol: UDP
-      port: 53
-  - to:
-    - podSelector:
-        matchLabels:
-          k8s-app: coredns
-    ports:
-    - protocol: UDP
-      port: 53
-    - protocol: UDP
-      port: 1053
-```
-
 ## Known issues and limitations
 
-- The upstream application only works with `kube-proxy`-based clusters. It will NOT work when using other `Kubernetes Service` implementations such as `Cilium` for example.
+- The upstream application only works with `kube-proxy`-based clusters. It will NOT work when using other `Kubernetes Service` (e.g. `cilium`). This app works with `cilium` by using `CiliumLocalRedirectPolicy` enabled by default.
```

Comment:
I'm not sure what information we are providing here. This is rather a configuration issue than a limitation.
In our case, with Cilium we followed this [guide](https://docs.cilium.io/en/stable/network/kubernetes/local-redirect-policy/#node-local-dns-cache). Other CNIs might need a different approach.
The upstream app is flexible enough to accommodate other mechanisms (it supports disabling the updates in iptables).

As I see it, non of the limitations described here are valid for our app (except the net-exporter one probably)

Reply:
Delete this point?

Reply:
```suggestion
- The upstream application only works with `kube-proxy`-based clusters. This app has been made compatible with `cilium` as the cluster network proxy following [this guide](https://docs.cilium.io/en/stable/network/kubernetes/local-redirect-policy/#node-local-dns-cache).

Reply:
I think it's worth noting how this app differs from upstream and how the functionality was achieved.